app/SwimChrono/.idea/misc.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
¿Por qué es necesario eliminar esta línea del código? ¿Se trata del encabezado del XML?

Reply:
misc.xml was uploaded in the repo and it's under .idea folder. It should be an Android studio dotfile, so we should clean the .idea folder from the repo